Some Fixes and Improvements

catIsBestAnimalIG opened the pull request:
- Fixed weird Layout Margins
- contacts are now easily customizable
- added poppins font
- added website link in readme

also can you add the website link in the about section

catIsBestAnimalIG committed:
- Fixed weird Layout Margins
- contacts are now easily customizable
- added poppins font
- added website link in readme

bederbourahmah-mof commented on src/components/Footer.tsx:
```diff
@@ -21,26 +23,28 @@ export default function Footer() {
+              target="_blank"
+              rel="noopener noreferrer"
+              className="text-gray-600 dark:text-gray-400 hover:text-gray-900 dark:hover:text-white transition-colors"
+              aria-label="Visit our GitHub repository"
```
I think this should also come from the object

bederbourahmah-mof commented:
@catIsBestAnimalIG thank you for your changes on this PR. All looks great. I like the new choice of font and we'll stick with it moving forward insha Allah unless there's a justification to use a different one, such as one that is more dyslexic friendly (I admittedly don't know which fonts are friendliest in that regard). I left one small comment on an issue I found with the `aria-label` being GitHub specific for every link. If you could please address that, then I can approve and merge. Thank you!

bederbourahmah-mof commented:
@catIsBestAnimalIG actually, one more thing, could you please add our LinkedIn https://www.linkedin.com/company/mosf ? I can add it after merging this if you'd prefer.

catIsBestAnimalIG committed:
fixed label and added linkedin

catIsBestAnimalIG committed:
fixed images

catIsBestAnimalIG committed:
added accidently removed files again

catIsBestAnimalIG commented:
I added linkedin in footer and fixed the label issue

Edit: also I fixed dark mode logo and header layout
<img width="1041" height="265" alt="Screenshot_20251124_153419" src="https://github.com/user-attachments/assets/42e3376c-cfef-4076-8e6f-cc8f767ca4f6" />
<img width="927" height="292" alt="Screenshot_20251124_153404" src="https://github.com/user-attachments/assets/f0883586-dc8e-4f34-b044-e9c4e5de4b14" />

catIsBestAnimalIG committed:
fixed logo and header layout and added a dark logo

BederBourahmah commented on public/images/logo-dark.png:
I like what you did here, but can you please make it so everything on the gear is filled in with white. These circles specifically.

Also, if you wanted access to all the raw files, including SVGs, you can find them here: https://github.com/muslimopensourcefoundation/mosf-assets/tree/main/logo%2Flogo-files

![Screenshot_20251124_065900_GitHub.jpg](https://github.com/user-attachments/assets/0297cc04-b6f8-48fb-b6ba-4044e60437e8)

BederBourahmah commented:
I will run again locally tonight insha Allah. I'm reviewing on my phone now and was comparing to production and realized that I didn't review impact on mobile. I'm sure it works well, I am just noting a gap in my review processes. I need a checklist 🤔

bederbourahmah-mof commented on public/images/logo-dark.png:
I could swear I left a comment on this when I was reviewing on my phone, but I guess it didn't go through. Can you please fill the little gray circles with white for consistency. Also, you can find all the raw assets here: https://github.com/muslimopensourcefoundation/mosf-assets/tree/main/logo/logo-files
<img width="361" height="388" alt="image" src="https://github.com/user-attachments/assets/4578caee-7727-4894-9589-26c550555faf" />

bederbourahmah-mof commented:
OK, all looks good. I tell you what, I'm going to merge as is and fix the minor dark mode logo issue on a separate PR.

bederbourahmah-mof reviewed (approved):
Looks good with the exception of the small inconsistency in the circles found on the logo. Will resolve promptly and a follow up PR.
